src/plugins/intel_cpu/src/edge.cpp

```diff
@@ -596,8 +596,10 @@ EdgePtr Edge::getBaseEdge(int look) {
         for (auto &ch_edge : ch_edges) {
             auto &chch_conf = ch_edge->getChild()->getSelectedPrimitiveDescriptor()->getConfig();
 
-            if (chch_conf.inConfs[ch_edge->getOutputNum()].inPlace() >= 0)
+            if (chch_conf.inConfs[ch_edge->getOutputNum()].inPlace() >= 0) {
+                break;
                 next_ch_edge = ch_edge;
+            }
```

Comment:
Sorry for the mistake, fixed in the new commit.

Reply:
it would be nicer If you can add more comments in the code explaining why we should select the first inplace consumer instead of last, this partof logic is pretty tricky, so comment would be helpful

Reply:
Add comments.